### ISSUE TITLE
KeystoneJS Slack

### ISSUE THREAD

**Issue by JedWatson** [keystonejs/keystone#534]
I've just created a [Slack](https://slack.com) Project for KeystoneJS, as a bit of an experiment to open a more direct line of communication between our growing team of contributors and collaborators.

I'd like to open it up to Keystone developers only at this point; so if you're hacking, extending, or building Keystone that's you! If you're using Keystone for a project, or are looking for help / support, please keep using our Google Group.

It's an invitation-based thing so if you'd like to join in, ping me an email (jed at keystonejs) or leave a comment here and I'll add you.

**Comment by JedWatson** [comment]
I think this issue has done its job - we have 38 excellent people (and counting) in the Slack team :smile: 

If anyone else wants in, let us know!

**Comment by mobinni** [comment]
Could I get an invite to your slack team? @jedwatson

**Comment by JedWatson** [comment]
@mobinni just sent you one!

**Comment by pswoodworth** [comment]
hey @JedWatson I'd love an invite too, if the slack channel is still going! my github username at gmail.

**Comment by JedWatson** [comment]
@pswoodworth invite sent :)

**Comment by niallobrien** [comment]
me at my Github username dot me

**Comment by tmvanetten** [comment]
@JedWatson  could I get an invite to the slack channel.  my Github username @ gmail

**Comment by JedWatson** [comment]
@tmvanetten sent!

**Comment by victoriafrench** [comment]
Could you add me as well please. I would like to help on #503. victoria dot french at cinecove dot com

**Comment by jneumann** [comment]
Can I join your reindeer games? I plan on trying to add some documentation. thejakeneumann at gmail dot com

**Comment by morenoh149** [comment]
@JedWatson I'm starting to think we should either destroy the slack of allow ppl to send themselves an invite

**Comment by tmvanetten** [comment]
@morenoh149 Don't destroy it.  It is useful.  If want someone to manage it I would volunteer to handle all the invites and such :)

**Comment by victoriafrench** [comment]
I agree with @morenoh149. I use slack to monitor the issues feed all day. Easier than opening and closing each and every issue on github.

**Comment by JedWatson** [comment]
@victoriafrench do you mean you agree with @tmvanetten?

@morenoh149 I'm a big fan of our slack, and find it great to have a direct channel with our contributors. I also think it serves a unique purpose, in that it's spammy to "chat" on github issues, and our gitter channel does a good job of handling community support (which is quite different to dev chat)

There's no specific requirements for entry other than being engaged with Keystone's development and wanting to help / participate. I agree the process for invitations could be streamlined, but want it to continue to be a request-based system and not an automatic signup.

In other words, you have to ask, but we've never said no :) it's a very friendly barrier to entry.

Maybe we can set up a simple form instead on the site that lets people submit a request, rather than using this issue, and it can go to a few volunteers who manage it (thanks @tmvanetten!)

Anyone interested in setting up that form?

**Comment by webteckie** [comment]
I think @morenoh149 is just citing with the CEO of Slack http://flip.it/RlsT-  :-)

**Comment by victoriafrench** [comment]
@JedWatson yes lol. I totally was probably drinking cool-aid while I wrote that due to my birthday party yesterday... I'm sure it was blurry as I read it... I'll stick with that. But I love slack.

**Comment by creynders** [comment]
@JedWatson maybe we can use this to streamline the onboarding of people in slack? https://github.com/rauchg/slackin

**Comment by suryagh** [comment]
Can I get an invite `surya.com at gmail`. Thank you.

**Comment by christroutner** [comment]
Can I get an invite? chris.troutner@gmail.com

**Comment by andreufirefly** [comment]
Can I get an invite please? Thanks

**Comment by tmvanetten** [comment]
@JedWatson if you can make me an admin on keystonejs slack team I will get automation with review set up.

**Comment by LorbusChris** [comment]
Could I get an invite, too? `c.glombek at gmail.com` Thanks :smile:

**Comment by JedWatson** [comment]
I've sent out all the invites, if anyone's asked for one and hasn't received it let me know!

**Comment by suryagh** [comment]
@JedWatson I have not received one: surya.com@gmail.com Thank you.

**Comment by jdponomarev** [comment]
@JedWatson could I get an invite? jdponomarev@gmail.com

**Comment by geloescht** [comment]
@JedWatson I'd like to discuss on how to contribute more and more efficiently to keystone, as it is quite an important matter for our product. Can I please get an invite to lukas at polyxo dot de ?

**Comment by mxstbr** [comment]
Can't believe you aren't already on there @geloescht...

**Comment by mxstbr** [comment]
I'm now an admin of the KeystoneJS Slack, invited @geloescht and @jdponomarev!

**Comment by r1b** [comment]
I would like an invite: robert.cole.jensen@gmail.com

**Comment by mxstbr** [comment]
Sent @r1b!

**Comment by geloescht** [comment]
It didn't seem to work for me. Can you please try lukas@polyxo.de again?

**Comment by mxstbr** [comment]
Resent it!

**Comment by misterdai** [comment]
I've love someone to throw me an invite: dave@yougeezer.co.uk

**Comment by r3wt** [comment]
gmorris_89@outlook.com very interested in contributing to the project. In particular i want to expand the capabilities and features of the generator.

**Comment by YvanDaSilva** [comment]
yvan.m.silva@gmail.com << Very interested in the project. Have been using it for a while, would love to contribute and talk with other devs.

**Comment by Vincent-Loi** [comment]
vincent@vincentloi.co.uk

Been developing and using keystone for a good few months now. Would love to join and contribute too! Shoot me an invite :)

**Comment by gorillatron** [comment]
jornandretangen at gmail.com :)

**Comment by justingreenberg** [comment]
@mxstbr would you mind adding me@justingreenberg.com as well when you have a sec, thanks!

**Comment by mattidupre** [comment]
Would love an invitation: matti@threebeluga.com   thanks / much appreciated

**Comment by radiovisual** [comment]
I would love an invitation to help with the development of keystone: senjudev@gmail.com

Thanks!

**Comment by mxstbr** [comment]
Sorry for the long wait, you're now all invited!

**Comment by michaelerobertsjr** [comment]
Can I get an invite michael.e.roberts.jr at gmail ? thx!

**Comment by jackoob** [comment]
Hi! I would be happy to be invited jaros.jakub at gmail . Thank you!

**Comment by bpavot** [comment]
Hi! Are you still inviting people ? I'd love to join benoit dot pavot at gmail
Thxs!

**Comment by blingerson** [comment]
ryan.p.mcginty at gmail. Thanks!

**Comment by ghuroo** [comment]
ghuroo at gmail dot com

thanks!

**Comment by arnibarnason** [comment]
@mxstbr I would like to join as well. arni at brimir.io. Thanks!

**Comment by panczarny** [comment]
hey, can you invite me? :D
czarny1906 at o2 dot pl

**Comment by icyretribution** [comment]
would love a chance to join,
icyretribution at gmail dot com

look forward to getting a chance to join the conversation!

**Comment by alexfaber2011** [comment]
I'd also enjoy being part of the conversation.  
alexfaber2011 at gmail dot com

Thanks much!

**Comment by iTsFILIPOficial** [comment]
I would love if you guys invited me to the convo
email: admin@itsfilipoficial.com

Thanks

**Comment by Elemino** [comment]
Hello, 

Interested in learning way more about Keystone. Hoping to join the slack channel! Cheers. 

elemshalit@gmail.com

**Comment by youbihi** [comment]
hey , I would like to help you guys , can you please add me on the slack group: youbihi at gmail dot com

Thanks

**Comment by dani190** [comment]
Can you add me... dc at danicela dot com

**Comment by vamshi9** [comment]
Could you please add me @JedWatson 
vamsisai999@gmail.com

**Comment by panczarny** [comment]
Hey, did anyone get invited? Cuz I wasn't :(

**Comment by christroutner** [comment]
I don't think I have the ability to send invites. But the Slack channel is
just as dead as this one.

-Chris
